homeassistant/components/unifiprotect/models.py

```diff
@@ -31,18 +40,26 @@ class PermRequired(int, Enum):
 class ProtectRequiredKeysMixin(EntityDescription, Generic[T]):
     """Mixin for required keys."""
 
-    ufp_required_field: str | None = None
-    ufp_value: str | None = None
+    ufp_required_field: tuple[str, ...] | str | None = None
```

Comment:
Maybe explain that the intention is to allow initializing instances with strings, but the instance attribute will always be tuples?